mqtt/mqtt-broker/Cargo.toml

```diff
@@ -68,8 +68,9 @@ name = "dispatch_messages"
 harness = false
 required-features = ["rand", "criterion", "__internal_broker_callbacks", "tokio/rt-multi-thread"]
 
-[features]
-benches = ["rand", "criterion", "__internal_broker_callbacks", "tokio/rt-multi-thread"]
+# This can be re-enabled once criterion updates rayon to 1.5.2 (rojang put a pr on rayon to fix security vulnerabilities)
+# [features]
+# benches = ["rand", "criterion", "__internal_broker_callbacks", "tokio/rt-multi-thread"]
```

Comment:
Even if this is disabled, criterion is still a dev-dep so it's still in the lockfile, so I assume CG will still complain?

Reply:
Also, I don't understand the comment. Your PR updates rayon's crossbeam dep from 0.8.0 to 0.8.1, but 0.8.1 is compatible with 0.8.0 so we don't need to wait for it. We can just `cargo update -p crossbeam-deque --precise 0.8.1` ourselves.

Reply:
Yeah, let me update this. The one thing I don't know on the CG front is what we need to do for test deps. For instance can we mark/tag certain deps to avoid getting security warnings? Or will it always be a "dismiss because test..."